modules/installation/pages/proxy-registration.adoc

```diff
@@ -3,7 +3,9 @@
 
 {productname} Proxy systems are registered as Salt or traditional clients using a bootstrap script.
 Migrating a traditionally registered Proxy system to a Salt Proxy system is not possible.
-Re-install the Proxy if you want to switch to Salt.
+Re-install the Proxy if you want to switch to Salt. Do not remove the old proxy from the list of
+registered systems and use the same activation key when reinstalling, so that you won't need to
+re-register the clients.
```

Comment:
```suggestion
Re-install the proxy if you want to switch to Salt.
When re-registering a proxy, do not remove the old proxy from the list of registered systems, and ensure you use the same activation key, to avoid having to re-register the clients.
```

I think this needs a whole lot more than these two sentences, but it's a start :woman_shrugging: